uds.js

```diff
@@ -106,6 +106,13 @@
         return executeUdsAjaxCall(onSuccess,onFailure,url,'GET');
     };
 
+    uds.fetchCaseAssociateDetails = function (onSuccess, onFailure, userId) {
+        if (!$.isFunction(onSuccess)) { throw 'onSuccess callback must be a function'; }
+        if (!$.isFunction(onFailure)) { throw 'onFailure callback must be a function'; }
+        var url =udsHostName.clone().setPath('/case/associates?where=roleName is "Contributor" and userId is "'+userId+'"');
```

Comment:
@renujhamtani can the "Contributor" role part be parametrized so as to pass any other role also ?